### PR TITLE
GSuite Scenario Testing Support

### PR DESCRIPTION
**Pull request by jacknagz**
### Background

More Cedar x Panther webinar assets

### Changes

* Update the send_data script to support GSuite Logs
* Write a script to convert JSONL logs to test_data YAML logs

### Testing

* YOLO (jk, we tested it for the webinar)

### CODE REVIEW

**Comment by jacknagz** [comment]
Stampy please @nhakmiller @lindsey-w @wey-chiang

**Review by lindsey-w — APPROVED** [review]
LGTM

File: test_scenarios/send_data.py [hunk 1]
```diff
@@ -105,14 +105,29 @@ def write_s3(client, bucket_name, logs, format):
...
+                event_time_format).replace(tzinfo=timezone.utc)
+
         log_event_time += event_time_shift
-        log[event_time_attr] = log_event_time.strftime(event_time_format)
+
+        if len(event_time_attrs) == 1:
+            log[event_time_attrs[0]] = log_event_time.strftime(event_time_format)
+        if len(event_time_attrs) == 2:
+            log[event_time_attrs[0]][event_time_attrs[1]] = log_event_time.strftime(event_time_format)
+
```
Comment (wey-chiang) [inline test_scenarios/send_data.py:128]:
```suggestion

```

File: test_scenarios/send_data.py [hunk 2]
```diff
@@ -105,14 +105,29 @@ def write_s3(client, bucket_name, logs, format):
...
     for log in logs:
-        log_event_time = datetime.strptime(log[event_time_attr], event_time_format).replace(tzinfo=timezone.utc)
+        if len(event_time_attrs) == 1:
+            log_event_time = datetime.strptime(
+                log[event_time_attrs[0]], event_time_format).replace(tzinfo=timezone.utc)
+        elif len(event_time_attrs) == 2:
+            log_event_time = datetime.strptime(
+                log[event_time_attrs[0]][event_time_attrs[1]],
+                event_time_format).replace(tzinfo=timezone.utc)
+
```
Comment (wey-chiang) [inline test_scenarios/send_data.py:121]:
```suggestion
        if len(event_time_attrs) == 1:
            log_event_time = datetime.strptime(
                log[event_time_attrs[0]], event_time_format).replace(tzinfo=timezone.utc)
            log[event_time_attrs[0]] = log_event_time.strftime(event_time_format)
        elif len(event_time_attrs) == 2:
            log_event_time = datetime.strptime(
                log[event_time_attrs[0]][event_time_attrs[1]],
                event_time_format).replace(tzinfo=timezone.utc)
            log[event_time_attrs[0]][event_time_attrs[1]] = log_event_time.strftime(event_time_format)

```

**Review by wey-chiang — APPROVED** [review]
Left a suggestion about combining redundant conditionals below but LGTM

**Pull request merged**